src/subaction_animations.jl

```diff
@@ -289,3 +289,63 @@ function _sethue(video, action, subaction, rel_frame)
     color = get_interpolation(subaction, rel_frame)
     Luxor.sethue(color)
 end
+
+"""
+    follow_path(points::Vector{Point}; closed=true)
+
+Can be applied inside a subaction such that the object defined in the parent action follows a path.
+It takes a vector of point which can be created as an example by calling `circle(O, 50)` <- notice that the action is set to `:none` the default.
+
+# Example
+```julia
+SubAction(1:150, follow_path(star(O, 300)))
+```
+
+# Arguments
+- `points::Vector{Point}` - the vector of points the object should follow
+
+# Keywords
+- `closed::Bool` default: true, sets whether the path is a closed path as for example when
+    using a circle, ellipse or any polygon. For a bezier path it should be set to false.
+"""
+function follow_path(points::Vector{Point}; closed = true)
+    (video, action, subaction, rel_frame) ->
+        _follow_path(video, action, subaction, rel_frame, points; closed = closed)
+end
+
+function _follow_path(video, action, subaction, rel_frame, points; closed = closed)
+    t = get_interpolation(subaction, rel_frame)
+    # if not closed it should be always between 0 and 1
+    if !closed
+        t = clamp(t, 0.0, 1.0)
+    end
+    # if t is discrete and not 0.0 take the last point or first if closed
+    if rel_frame != 1 && isapprox_discrete(t)
```

Comment:
the path you use is two points so `A` and `B` but the default kwarg is `closed = true` to make it easy to work with polygons basically. This means the movement is `A -> B -> A`. This basically checks: okay we are not in the first frame but we are discrete therefore we want to just return the end point instead of continuing with the computation of finding the point.

Reply:
This makes sense! Thanks for explaining.